linkedList.js

```diff
@@ -4,5 +4,79 @@
  * @return {Object} an object exposing methods to be used to manipulate a linked list
  */
 function linkedListGenerator(){
+    let head = null;
+    let tail = null;
+    return {
+        getHead: function(){
+            return head;
+        },
+        getTail: function(){
+            return tail;
+        },
+        add: function(value){
+            let newNode = new Object;
+            newNode.value = value;
+            newNode.next = null;
+            if (!head){
+                head = newNode;
+                tail = newNode;
+            } else {
+                tail.next = newNode;
+                tail = newNode;
+            }
+            return newNode;
+        },
+        get: function(index){
+            let current = head;
 
+            let i = 0;
+            while (current !== null) {
+                if (i === index) {
+                    return current;
+                } else { 
+                    current = current.next;
+                    i++;
+                }
+            }
+            return false;
+         
+        },
+        remove: function(index){
+            let current = this.get(index);
+            let before = this.get(index - 1);
+            if (current === false){
+                return false;
+            }
+            if (index === 0){
+                head = current.next;
+                before = false;
```

Comment:
you don't need the before anymore, it's not going to be used anywhere after this.